### PR TITLE
Secure machine fixes + customer feedback 

### PR DESCRIPTION
**Pull request by vanmaegima**
## Readiness

* [ ] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by vanmaegima** [comment]
@doanac can you please review the changes to aktualizr-lite.rst?

**Review by angolini — APPROVED** [review]

File: source/reference-manual/security/secure-boot-imx-ahab.rst [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 .. _ref-secure-boot-imx-ahab:
 
-Secure Boot on i.MX 8 and i.MX 8X Families using AHAB
-=====================================================
+Secure Boot on i.MX 8/8X Families using AHAB
```
Comment (Tim-Anderson) [inline source/reference-manual/security/secure-boot-imx-ahab.rst:5]: It is not clear here or in the NXP documentation that 8QM is in this category.  You have to dig into documentation in NXP to see that this includes 8QM. We might want to just add `including 8QM` in the title or first line.

File: source/reference-manual/security/secure-boot-imx-ahab.rst [hunk 2]
```diff
@@ -54,10 +57,10 @@ can be inspected by executing the ``print_fuses`` script in that same directory.
 
 The Security Reference Manual for your specific SoC indicates which fuses need to be programed with the SRK fuse information.
 
-i.MX8QM fusing
+i.MX 8QM fusing
 --------------------------
 
-On the i.MX8QM SoC fuses are stored in fuse bank 0, words 722 to 737::
+On the i.MX 8QM SoC fuses are stored in fuse bank 0, words 722 to 737::
```
Comment (Tim-Anderson) [inline source/reference-manual/security/secure-boot-imx-ahab.rst:63]: We need to make it clear that these are example values and not meant for production.

**Review by Tim-Anderson — COMMENTED** [review]
Commented on recommended additions

File: source/reference-manual/security/secure-boot-imx-ahab.rst [hunk 3]
```diff
@@ -76,7 +79,7 @@ On the i.MX8QM SoC fuses are stored in fuse bank 0, words 722 to 737::
         => fuse prog -y 0 736 0x6E0B791C
         => fuse prog -y 0 737 0x6A558134
 
-Alternatively, use the kernel to program SoC fuses using SDP via NXP's Universal Update Utility with a script as follows::
+Alternatively, you can program SoC fuses via SDP using NXP's Universal Update Utility with a script as follows::
```
Comment (kprosise) [inline source/reference-manual/security/secure-boot-imx-ahab.rst:82]:
```suggestion
Alternatively, you can program SoC fuses via SDP  by using NXP's Universal Update Utility. This is shown in the following script::
```

**Review by kprosise — APPROVED** [review]
Overall it LGTM, a couple suggestions to increase clarity.

File: source/reference-manual/ota/aktualizr-lite.rst [hunk 4]
```diff
@@ -40,10 +40,12 @@ To view the daemon logs:
 Manual Mode
 -----------
 
-If you have disabled daemon mode, you can use ``aktualizr-lite`` manually from the command line to fetch updates and apply them. Manual mode can be useful for debugging, testing, or demoing a device.
+Disabling daemon mode is not recommended, but running ``aktualizr-lite`` manually can be useful for debugging, testing, or demoing a device.
 
 .. note:: Manual mode will require you to reboot your device to apply an update.
 
+.. note:: If ``aktualizr-lite`` default daemon mode does not fit your needs, the alternative is to create a :ref:`ug-custom-sota-client`.
+
```
Comment (vanmaegima) [inline source/reference-manual/ota/aktualizr-lite.rst:48]: @doanac can you have a look at this change?

Comment (doanac) [reply source/reference-manual/ota/aktualizr-lite.rst:48]: that's a good idea. +1

**Pull request merged**